implemented get_powermeters()

bullitt186 opened the pull request:
As discussed in [PR 79](https://github.com/fsantini/python-e3dc/pull/79) with @torbennehmer and @vchrisb, here's an implementation of "get_powermeters()" which returns a dictionary of available powermeters (index + type).
This object can be right away be used when using get_powermeters_data() or as part of the config object.

vchrisb commented:
This package https://github.com/spali/go-rscp/blob/master/rscp/tag.go#L561 has a very good insight in what the returned values actually are.
Beside fixing the lint issues, it would be great to incorporate the parsing of the returned int to the human readable string.
This could then also be used in `get_powermeter_data` 
Thank you!

bullitt186 commented:
Thank you for your feedback, i fixed the linting issues and introduced besides ``type`` now ``typeName`` which includes the powermeter type according to your source.
There are manifold ways how you could translate the type id to strings, but i couldn't find a best practice in the project so far. So I hope my way works for you.

vchrisb commented:
Would you mind moving the type dict here: https://github.com/fsantini/python-e3dc/blob/master/e3dc/_rscpTags.py#L2092
This file contains all the different tags and it could be reused.
Also use hex values for consistency. 
Thank you for your effort. 👍

bullitt186 commented:
Thank you for your patience and guiding me!

I moved the powermetertypes to a data structure ``powermeterTypes``within ``_rscpTags.py``. As the types do not identify rscp specific messages / data fields, i did not prefix the data structure with "rscp". I also added a ``getPowermeterType()``similar to ``getDatatype()``.
To access this method, i had to include to add ``from ._rscpTags import getPowermeterType``in ``_e3dc.py``.
This seemed better to me than to add another wrapper within ``rscpLib.py``

vchrisb commented on e3dc/_e3dc.py:
```diff
@@ -1746,6 +1747,47 @@ def get_pvis_data(self, pvis=None, keepAlive=False):
+                        ("PM_REQ_TYPE", "None", None),
+                    ],
+                ),
+                keepAlive=keepAlive,
```
Due to the loop, the consecutive calls should have `keepAlive=True`, and only the last call should honor the passed keepAlive,
Have a look at https://github.com/fsantini/python-e3dc/blob/master/e3dc/_e3dc.py#L1312
